### DISCUSSION TITLE
Add support for the second solar register "se2"

### DISCUSSION THREAD

**Discussion by WahlFisch91** [dominikamann/oekofen-pellematic-compact#51] [Ideas]
Hi,
is it possible to add support for the second solar register named "se2" in the http API?
I have only one solar register, but the Pellematic Compact API shows me my data at the point "se2", for any reason...

Thanks!
LG Wolfgang

**Comment by dominikamann** [comment]
Hi, 

see https://github.com/dominikamann/oekofen-pellematic-compact/releases/tag/v3.2.0
You need to delete the current heater and add it again. 

LG,
Dominik

**Comment by WahlFisch91** [comment]
Hi Dominik,

 

Thanks for your changes! - I really appreciate that!

Could you please set the "sk" data field to a variable field to? - so i can enter my "sk3" circuit? (See Screenshot attached)

 

 

 

Thanks a lot!
 

Mit freundlichen Grüßen,
best regards,

Wolfgang Wahl

 
 

Gesendet: Mittwoch, 22. Mai 2024 um 10:53 Uhr
Von: "Dominik Amann" ***@***.***>
An: "dominikamann/oekofen-pellematic-compact" ***@***.***>
Cc: "WahlFisch91" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [dominikamann/oekofen-pellematic-compact] Add support for the second solar register "se2" (Discussion #51)


 

Hi,

see https://github.com/dominikamann/oekofen-pellematic-compact/releases/tag/v3.2.0

**Comment by dominikamann** [comment]
In your first comment you only wrote about SEx. Is it now also SK? Are there any other areas that need to be flexible? Doing it step by step takes a long time ...

**Reply by WahlFisch91** [reply]
sorry for that - did not noticed that in my first view...
that would be all, regarding my pellematic configuration.

**Comment by dominikamann** [comment]
https://github.com/dominikamann/oekofen-pellematic-compact/releases/tag/v3.2.1

Dominik
